sycl/test-e2e/ESIMD/fp_controls.cpp

```diff
@@ -0,0 +1,240 @@
+// REQUIRES: gpu && (opencl || level_zero)
```

Comment:
minor comment: this line is not needed because we have this file: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/lit.local.cfg
```suggestion
```

Reply:
Removed, thanks!